composer.json

```diff
@@ -124,7 +124,6 @@
         "doctrine/data-fixtures": "^1.1",
         "doctrine/dbal": "^2.6|^3.0",
         "doctrine/orm": "^2.6.3",
-        "doctrine/doctrine-bundle": "^1.5|^2.0",
```

Comment:
Apparently, we relied on DoctrineBundle to pull `doctrine/annotations` for us. That package should be added instead.

Reply:
updated